doc/man7/EVP_MAC-KMAC.pod

```diff
@@ -39,12 +39,12 @@ EVP_MAC_CTX_get_params(), or with EVP_MAC_size().
 
 =item B<OSSL_MAC_PARAM_SIZE> ("size") <size_t>
 
-=item B<OSSL_MAC_PARAM_XOF>
```

Comment:
This should be _int_, not _integer_.  These are C types.

Reply:
No.  The *implementation* will most probably read it as an int, but from a param perspective, it's a `OSSL_PARAM_INTEGER`.

Or, seen differently, how is `octet string` a C type?

Reply:
I am going to leave it as an integer, as that is what it uses in EVP_MAC.pod..